e2e/terraform/packer/ubuntu-bionic-amd64/provision.sh

```diff
@@ -195,6 +202,12 @@ if [ -n "$CONSUL_AUTOJOIN" ]; then
     update_consul_autojoin
 fi
 
+if [ -n "$NOMAD_LICENSE" ]; then
+  sudo touch /etc/nomad.d/.environment
```

Comment:
I think this line should be before the conditional so that the file is there even if `NOMAD_LICENSE` isn't set?

Reply:
:+1: sounds good